apps/perf-test/src/scenarios/ChoiceGroupNext.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
```

Comment:
The main goal of doing this now is to ensure we didn't regress perf by FC conversion (given hook calls are not cheap).
Ideally we have perf tests for ALL components (i believe that has been always something we want to do but never got to). Couple things to also take into consideration: 
1) perf tests today do not blocking build
2) given 1), it doesn't mean it can be slow. if we need to scale the perf test, do that that, without limiting our ability to write new tests

All this being said, I am open to discussion. For this particular case, I agree it doesn't seem to be very useful. Perhaps given the goal here, what we can do instead is to do a perf run before we ship (v7 vs. v8 component)